src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { AppBar, Typography } from '@mui/material';
+import Link from 'next/link';
+import { SignOutButton } from '../Buttons/Buttons';
+import { getServerSession } from 'next-auth/next';
+import { authOptions } from '@/lib/auth';
+
+export default async function NavBar() {
+  const session = await getServerSession(authOptions);
+
+  return (
+    <AppBar
+      position="static"
+      sx={{
+        position: 'static',
```

Comment:
Is there a reason why `position="static"` is defined twice? If not, I would remove one of the definitions

Reply:
No reason. Just forgot that there..